flink-connectors/flink-connector-rabbitmq2/README.md

```diff
@@ -0,0 +1,24 @@
+# License of the Rabbit MQ Connector
```

Comment:
nit: What about merging `README.md` of RMQSource and `README.md` of RMQSink into this `RMQSource` ?